driver/ppm_events_public.h

```diff
@@ -951,15 +956,23 @@ enum ppm_event_type {
 	PPME_SYSCALL_LINK_2_X = 309,
 	PPME_SYSCALL_LINKAT_2_E = 310,
 	PPME_SYSCALL_LINKAT_2_X = 311,
-	PPME_SYSCALL_FCHMODAT_E = 312,
-	PPME_SYSCALL_FCHMODAT_X = 313,
-	PPME_SYSCALL_CHMOD_E = 314,
-	PPME_SYSCALL_CHMOD_X = 315,
-	PPME_SYSCALL_FCHMOD_E = 316,
-	PPME_SYSCALL_FCHMOD_X = 317,
-	PPME_SYSCALL_RENAMEAT2_E = 318,
-	PPME_SYSCALL_RENAMEAT2_X = 319,
-	PPM_EVENT_MAX = 320
+	PPME_SYSCALL_CHOWN_E = 312,
+	PPME_SYSCALL_CHOWN_X = 313,
+	PPME_SYSCALL_LCHOWN_E = 314,
+	PPME_SYSCALL_LCHOWN_X = 315,
+	PPME_SYSCALL_FCHOWN_E = 316,
+	PPME_SYSCALL_FCHOWN_X = 317,
+	PPME_SYSCALL_FCHOWNAT_E = 318,
+	PPME_SYSCALL_FCHOWNAT_X = 319,
+	PPME_SYSCALL_CHMOD_E = 320,
+	PPME_SYSCALL_CHMOD_X = 321,
+	PPME_SYSCALL_FCHMOD_E = 322,
```

Comment:
fchmod already had a position, it will need to be kept there for scap compatibility.

Reply:
I'm not sure what you mean by that, should I keep the previous numbering for the one that already exists?